src/Queen.cpp

```diff
@@ -1 +1,16 @@
-#include "Rook.h"
\ No newline at end of file
+#include "Queen.h"
+#include "FactoryPieces.h"
+
+bool Queen::m_registerit = FactoryPieces::registerit(QUEEN_CHAR, [](const char& _ch, const Location& loc, PlayerColor color, Board* board)
+	-> unique_ptr<Piece> {return std::make_unique<Queen>(_ch, loc, color, board); }
+);
+
+Queen::Queen(const char& _ch, const Location& loc, PlayerColor color, Board* board)
+	: Piece::Piece(_ch, loc, color, board), Bishop(_ch, loc, color, board), Rook(_ch, loc, color, board) 

```

Comment:
there is no need to call the Piece::Piece() constructor explicitly since it is called implicitly. So, you can remove the Piece:: prefix before the constructor call.